composer.json

```diff
@@ -136,9 +136,11 @@
     "drupal/encrypt": "^3.0@alpha",
     "drupal/jira_rest": "^2.0",
     "drupal/key": "^1.7",
-    "drupal/real_aes": "^2.1"
+    "drupal/real_aes": "^2.1",
```

Comment:
As a stickler bit of feedback, it is really helpful when this list is in alphabetical order. Makes life easier for the next one to find the thing they are looking for.

Reply:
🙄 haha